docs/11-circleci/02-activation.mdx

```diff
@@ -0,0 +1,135 @@
+---
+toc_max_heading_level: 4
+---
+
+# Activation
+
+The orb uses a Unity Editor instance to test and build your projects. These operations require the
+instance to be activated before each job you run.
```

Comment:
This part I just read in the previous document. Let's leave it away from this file to make it more concise.

Reply:
I'm repeating this here because the users won't always read the sections in order like you did. I plan to link the Activation section in a couple of error messages in the orb. And users coming from these messages will miss that informational context if I restrict it to the Getting Started section.

Let me know what you think.

Reply:
Right. Or perhaps we should just leave it away from the getting started section.

I'm good with whatever you choose to do here.

Reply:
IIRC, I condensed the information on Getting Started based on another suggestion you made. I think we are good now.